### PR TITLE
Transfers related code cleanup

### PR DESCRIPTION
**Pull request by GPaoloni**
- Removed various functions that are not longer necessary
- Simplified (and unified) the logic behind who controls a task

### CODE REVIEW

File: plugin-hrm-form/src/utils/transfer.js [hunk 1]
```diff
@@ -124,11 +117,15 @@ export const clearTaskControl = async task => {
 export const updateTransferStatus = newStatus => async task => {
   const updatedAttributes = {
     ...task.attributes,
+    transferStarted: true,
     transferMeta: {
       ...task.attributes.transferMeta,
       transferStatus: newStatus,
+      /*
```
Comment (nickhurlburt) [inline plugin-hrm-form/src/utils/transfer.js:124]: Why is this here?

Comment (GPaoloni) [reply plugin-hrm-form/src/utils/transfer.js]: Sorry it should't be there, removed already.
